tekton/utils/pipeline_ref_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	// . "github.com/onsi/gomega/gstruct"
+)
+
+var _ = Describe("Release type", func() {
+	var (
+		clusterPpr PipelineRef
+		gitPpr     PipelineRef
+		bundlePpr  PipelineRef
+	)
+
+	BeforeEach(func() {
```

Comment:
I'd declare them in each test in this particular case.

Reply:
In this case it would be exactly the same definition done twice, so I think it makes to do it just once. Please check.